common/lib/capa/capa/tests/test_html_render.py

```diff
@@ -151,10 +151,6 @@ def test_render_response_xml(self):
         # Expect problem has been turned into a <div>
         self.assertEqual(rendered_html.tag, "div")
 
-        # Expect question text is in a <p> child
-        question_element = rendered_html.find("p")
```

Comment:
The question text is no longer present in the rendered html because it is added in the response element. And that callback is mocked, so all that we can check is that it is passed in the context (see line 184).